src/mbgl/renderer/paint_parameters.cpp

```diff
@@ -139,6 +145,11 @@ void PaintParameters::clearStencil() {
                          0,
                          0}};
     mtlContext.renderTileClippingMasks(*renderPass, staticData, tileUBO);
+    context.renderingStats().stencilClears++;
+#elif MLN_RENDER_BACKEND_VULKAN
+    const auto& vulkanRenderPass = static_cast<vulkan::RenderPass&>(*renderPass);
+    vulkanRenderPass.clearStencil();
```

Comment:
`vulkan::RenderPass::clearStencil` is the fullscreen version that uses the framebuffer attachments and since it needs a valid render pass/framebuffer/command buffer it was the safest place to enforce these conditions.

Reply:
@mwilsnd Could you resolve this conversation if you are happy with that answer? 😇